src/EA.Weee.Web/Areas/Admin/Controllers/ManageEvidenceNotesController.cs

```diff
@@ -85,6 +90,7 @@ public async Task<ActionResult> ViewEvidenceNote(Guid evidenceNoteId)
         }
 
         [HttpGet]
+        [AuthorizeInternalClaims(Claims.InternalAdmin)]
```

Comment:
We should remove this attribute from the action as any internal user can view transfer notes.